release.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/sh
```

Comment:
this script is our actual release stuff - all the things noted in the PR description.